demonstrations/HHL_with_catalyst_qrisp.metadata.json

```
@@ -0,0 +1,48 @@
+{
+    "title": "HHL with Qrisp x Catalyst",
+    "authors": [
+        {
+            "username": "renezander"
+        },
+        {
+            "username": "positr0nium"
+        },
+        {
+            "username": "matic_p"
+        }
+    ],
+    "dateOfPublication": "2025-02-11T09:00:00+00:00",
+    "dateOfLastModification": "2025-02-11T09:00:00+00:00",
+    "categories": [
+        "How-to",
+        "Algorithms"
+    ],
+    "tags": [],
+    "previewImages": [
+        {
+            "type": "thumbnail",
+            "uri": "/_static/demo_thumbnails/regular_demo_thumbnails/thumbnail_HHL_with_catalyst_qrisp.png"
+        },
+        {
+            "type": "large_thumbnail",
+            "uri": "/_static/demo_thumbnails/large_demo_thumbnails/thumbnail_large_HHL_with_catalyst_qrisp.png"
+        }
+    ],
+    "seoDescription": "SEO description to be filled in (TBD)",
+    "doi": "",
+    "references": [
+        {
+            "id": "Draper2000",
+            "type": "preprint",
+            "title": "Addition on a Quantum Computer",
+            "authors": "Thomas G. Draper",
+            "year": "2000",
+            "doi": "10.48550/arXiv.quant-ph/0008033",
+            "url": "https://arxiv.org/abs/quant-ph/0008033"
+        }
+    ],
+    "basedOnPapers": [],
+    "referencedByPapers": [],
+    "relatedContent": [
+    ]
```

These 3 demos may be good candidates for 'related materials' section (note, we can add up to 3).

```suggestion
    "relatedContent": [
        {
            "type": "demonstration",
            "id": "tutorial_qjit_compile_grovers_algorithm_with_catalyst",
            "weight": 1.0
        },
        {
            "type": "demonstration",
            "id": "tutorial_how_to_quantum_just_in_time_compile_vqe_catalyst",
            "weight": 1.0
        },
        {
            "type": "demonstration",
            "id": "tutorial_qpe",
            "weight": 1.0
        }
    ]
```

---

Great suggestions!